cfn-resources/auditing/cmd/resource/resource.go

```diff
@@ -38,6 +39,7 @@ func setup() {
 
 func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler.ProgressEvent, error) {
 	setup()
+	util.SetDefaultProfileIfNotDefined(&currentModel.Profile)
```

Comment:
lines just below that handle defining the profile value can be removed, applies for other operations as well

Reply:
updated